docs/readthedocs/source/doc/LLM/Quickstart/docker_cpp_xpu_quickstart.md

```diff
@@ -0,0 +1,145 @@
+## RUN llama.cpp and ollama and open-webui with IPEX-LLM CPP docker image on Intel GPUs
+
```

Comment:
please orgnize the doc as follows：

Install - refer links to install guide
Launch container - pull/run specific image
Verify performance by running perf benchmark
Run llama.cpp
Run Ollama models (interactive)
Run Ollama serve
Run open-webui